transport/src/main/java/io/netty/channel/AbstractChannelHandlerContext.java

```diff
@@ -153,7 +150,10 @@ public ChannelHandlerContext fireUserEventTriggered(Object event) {
     @Override
     public ChannelHandlerContext fireChannelRead(Object msg) {
         AbstractChannelHandlerContext next = findContextInbound();
-        ReferenceCountUtil.touch(msg, next);
+        if (ResourceLeakDetector.ENABLED) {
```

Comment:
@ninja- - I guess don't worry about it because @normanmaurer will take over this issue (and submit a new PR).